index.html

```diff
@@ -1731,16 +1701,11 @@
         The set of presentations known to the user agent should be cleared when
         the user requests to "clear browsing data."
       </p>
-      <p class="open-issue">
-        The spec should specify any restrictions on the presenting browsing
-        context when the opening browsing context is in "incognito" mode. See
-        <a href="https://github.com/w3c/presentation-api/issues/14">ISSUE 14:
-        Define user agent context for rendering the presentation</a>
-      </p>
-      <p class="open-issue">
+      <div class="issue" data-number="14"></div>
```

Comment:
Updated https://github.com/w3c/presentation-api/issues/14 with the removed text.